dask_sql/physical/rel/logical/sort.py

```diff
@@ -21,7 +20,10 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
         (dc,) = self.assert_inputs(rel, 1, context)
         df = dc.df
         cc = dc.column_container
-
+        sort_expressions = rel.sort().getCollation()
+        sort_columns = [expr.column_name(rel) for expr in sort_expressions]
```

Comment:
The newly added `column_name` method for `PyExprs` seems to be failing here with 
```python
RuntimeError: Internal("Create name does not support sort expression")
```

I'm not sure if some changes need to be made to this helper method or if the sort [variant](https://docs.rs/datafusion/latest/datafusion/logical_plan/enum.Expr.html#variant.Sort) needs to be matched separately and the `expr` member needs to be calling this method. cc: @andygrove

Reply:
@ayushdg  There are a couple of different ways we can resolve this but I think I need to understand the context better first. How will these column names be used? If we get a name such as `first_name ASC NULLS FIRST` is that what you are looking for, or are you just looking for the underlying column name?

Reply:
The goal is to get the underlying column name and use that information with the ascending and Nulls first ordering to perform a sort on the Dask DataFrame layer. Since we're already getting the other 2 attributes from the helper function included in the pr, the only thing needed here is the underlying column names we're sorting on

Reply:
Could you try the solution in https://github.com/dask-contrib/dask-sql/pull/515 and see if that resolves this?

Reply:
Yup that does the trick. Thanks!